### ISSUE TITLE
How do you deal with responsive typography?

### ISSUE THREAD

**Issue by woosaj** [HugoGiraudel/ama#73]
Hello, I would like to know how do you deal with responsive typography? Do you use rem units, ems, or px? 

How do you scale typography? Maybe you set 100% on `html` tag and then on different breakpoints you change that value? Or you use modular scale?

Thanks in advance.

**Comment by HugoGiraudel** [comment]
Hey,

Wow, I completely forgot this topic, I am very sorry. I should totally have set me a reminder!

Okay so I’m not the best person to ask about responsive typography as I tend to keep things very simple. Usually I just set a font-size in `em` on the `body` element, as well as a unitless line-height.

From there, I use `em` all over the place (or rather `%` as I find them more explicit for `font-size`) to rely on the cascade for dynamic font sizing. I never change the font size globally when going from one breakpoint to another. I never found that necessary.

However note that I never worked on projects where finely grained typography was tremendously important. That’s why I might not be the best person to ask. 

Sorry for not being able to help any further. :)

**Comment by Zhouzi** [comment]
@zellwk has written some great in-depth articles on the subject: https://zellwk.com/blog/

@HugoGiraudel and what's your opinion regarding `rem`s? Don't you find `em`s hardly predictable? Or is it just about some kind of mindset?

**Comment by woosaj** [comment]
@HugoGiraudel thanks for answer, better late than never :)

I'm going to use Typi on the next project and will see how it goes.

**Comment by zellwk** [comment]
I hope I'm not intruding the thread, but I got an email since @Zhouzi mentioned me up above. Let me know if I am, and I'll delete this comment. 

@Zhouzi I have an article on REM vs EM if it helps any at https://zellwk.com/blog/rem-vs-em/
@woosaj Let me know how Typi works out for ya. Just send me an email if you have questions. You might also be interested in these articles: 
- https://zellwk.com/blog/responsive-typography/
- https://zellwk.com/blog/why-vertical-rhythms/

**Comment by HugoGiraudel** [comment]
>  and what's your opinion regarding rems? Don't you find ems hardly predictable? Or is it just about some kind of mindset?

I think it essentially boils down to mindset and the type of project you are working on. If you are building around highly dynamic and unpredictable user generated content (CMS, templates, etc.), avoiding `em` might be an idea. If you build independent modular components, I believe there is little harm that could ever be done.

Therefore I never use `rem`. I used to when it started being supported and it was hype and all, a few years ago, but I realised I don’t really need them. I’d love to be proven wrong though. As I said before, scaling typography has never been my strong suit;
